app/controllers/organizations_controller.rb

```diff
@@ -229,4 +229,29 @@ def get_inactive_days(membership, period_start)
                end
     days_off.to_i
   end
+
+  def get_members_inactive_days
+    return unless @team
+
+    inactive_days = {}
+    period_start = @month_limit.nil? ? Time.current - 9.months : Time.current - @month_limit.month
+    @team.github_users.each do |user|
+      membership = FroggoTeamMembership.find_by(github_user: user, froggo_team: @team)
+      inactive_days[user.id] = get_inactive_days(membership, period_start)
+    end
+    inactive_days
+  end
+
+  def get_inactive_days(membership, period_start)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Lint/DuplicateMethods: Method OrganizationsController#get_inactive_days is defined at both app/controllers/organizations_controller.rb:220 and app/controllers/organizations_controller.rb:245.